doc/configuration.rst

```diff
@@ -143,6 +143,25 @@ You can specify the following options in module configuration.
         position = 'center'
     }
 
+.. note::
+    New in version 3.16
+
+You can specify the options in module or py3status configuration section.
+
+The following options will work on ``i3bar-gaps``.
+
+``urgent_background``: Specify urgent background color for modules.
+``urgent_foreground``: Specify urgent foreground color for modules.
+``urgent_border``: Specify urgent border color for modules.
```

Comment:
the first three options do work on i3bar so please make the distinction

Reply:
Done. No issue in rebasing. Just had to fix docs and that was mostly it.